Source/Api/Controllers/EventController.cs

```diff
@@ -625,6 +625,15 @@ public class EventController : RepositoryApiController<IEventRepository, Persist
 
             return Ok();
         }
+        
+        [HttpPost]
+        [Route("~/entries")]
```

Comment:
It's the end point they submit too, if we match their api, then it would automatically be handled. Would be interested in helping @skoub finish this pr? I'd be more than happy to meet up with you two and merge in the latest changes.